Knoema.Client/Meta/ResultStatus.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using System.Collections.Generic;
 
-namespace Knoema.Upload
+namespace Knoema.Meta
 {
-	public class VerifyDatasetResult
+	public class ResultStatus
```

Comment:
Why do you change the names of classes and namespaces? it will break backward compatibility.

Reply:
restored class & filename